### ISSUE TITLE
Wrong BlacklistTask output

### ISSUE THREAD

**Issue by sc0rp10** [phpro/grumphp#293]
| Q               | A
| --------------- | ---
| Branch          | master
| Bug?            | yes

Hi!
I have legacy project with some codegenerating classes with ugly code inside.
And I have blacklisted keyword 'private $' for prevent creating private field (in favor of protected).
When I make some changes in classes with codegenerating, i'm got ugly formatted error about my blacklisted word:
```
t.php
  1:14:  null;
```
which is less informative: it contains wrong line/symbol number and cutted and unreadable description.

**My configuration**
```yaml
# grumphp.yml
parameters:
    tasks:
      git_blacklist:
        keywords:
          - "private \\$"
        triggered_by: [php]
```

`cat t.php`
```php
<?php

class F
{
    protected function foo()
    {
        $code = <<<PHP
    private $' . $this->callSomeVeryVeryVeryLongMethodName() . ' = null;
PHP;
    }
}

```
# Result
```
# Run GrumpHP:
$ git ci -m "init"
GrumPHP detected a pre-commit command.
GrumPHP is sniffing your code!
Running task 1/1: Blacklist
# skip ascii
You have blacklisted keywords in your commit:
t.php
  1:14:  null; # <- wut?
To skip commit checks, add -n or --no-verify flag to commit command
```

**Comment by veewee** [comment]
I would suggest using a custom visitor in combination with the phpparser task for this kind of checks.
The blacklist is made for easy matches. If you want to check more advanced patterns, the phpparser is the way to go.

Nevertheless, we should fix the output of the blacklist task.

**Comment by veewee** [comment]
Fixed in #375
